### PR TITLE
Update every apigouvlink & fix IG 400 for not luhn valid siren

### PR DESCRIPTION
**Pull request by XavierJp**
title

### CODE REVIEW

File: app/(header-default)/donnees/api-entreprises/page.tsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ export default function AccesByAPIPage() {
       <ul>
         <li>
           Conditions d’accès :{' '}
-          <a href="https://api.gouv.fr/les-api/api-entreprise/demande-acces">
+          <a href="https://www.data.gouv.fr/fr/dataservices/api-entreprise/demande-acces">
```
Comment (HAEKADI) [inline app/(header-default)/donnees/api-entreprises/page.tsx:84]: This link does not work.

File: app/(header-default)/donnees/api-entreprises/page.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ export default function AccesByAPIPage() {
       <ul>
         <li>Conditions d’accès : API ouverte</li>
         <li>
-          <a href="https://api.gouv.fr/les-api/api-recherche-entreprises">
+          <a href="https://www.data.gouv.fr/fr/dataservices/api-recherche-entreprises">
```
Comment (HAEKADI) [inline app/(header-default)/donnees/api-entreprises/page.tsx:43]:
```suggestion
          <a href="https://www.data.gouv.fr/fr/dataservices/api-recherche-dentreprises/">
```

File: app/(header-default)/donnees/api-entreprises/page.tsx [hunk 3]
```diff
@@ -103,7 +103,11 @@ export default function AccesByAPIPage() {
       </p>
       <ul>
         <li>
-          l’<a href="https://api.gouv.fr/les-api/sirene_v3">API Sirene</a> de l’
+          l’
+          <a href="https://www.data.gouv.fr/fr/dataservices/api-sirene">
```
Comment (HAEKADI) [inline app/(header-default)/donnees/api-entreprises/page.tsx:107]:
```suggestion
          <a href="https://www.data.gouv.fr/fr/dataservices/api-sirene-1/">
```

File: README.md [hunk 4]
```diff
@@ -12,7 +12,7 @@
 <a href="https://annuaire-entreprises.data.gouv.fr/historique-des-modifications"><img src="https://img.shields.io/badge/Page-changelog-blue.svg" alt="Changelog Badge"></a>
 <a href="https://annuaire-entreprises.data.gouv.fr/a-propos/stats"><img src="https://img.shields.io/badge/Page-stats-blue.svg" alt="Statistiques Badge"></a>
 
-Dépôt du site [L'Annuaire des Entreprises](https://annuaire-entreprises.data.gouv.fr). Pour [l’API Recherche d’Entreprises](https://api.gouv.fr/les-api/api-recherche-entreprises), consultez le [dépôt de l’API](https://github.com/annuaire-entreprises-data-gouv-fr/search-api).
+Dépôt du site [L'Annuaire des Entreprises](https://annuaire-entreprises.data.gouv.fr). Pour [l’API Recherche d’Entreprises](https://www.data.gouv.fr/fr/dataservices/api-recherche-entreprises), consultez le [dépôt de l’API](https://github.com/annuaire-entreprises-data-gouv-fr/search-api).
```
Comment (HAEKADI) [inline README.md:15]:
```suggestion
Dépôt du site [L'Annuaire des Entreprises](https://annuaire-entreprises.data.gouv.fr). Pour [l’API Recherche d’Entreprises](https://www.data.gouv.fr/fr/dataservices/api-recherche-dentreprises/), consultez le [dépôt de l’API](https://github.com/annuaire-entreprises-data-gouv-fr/search-api).
```

File: app/(header-default)/a-propos/comment-ca-marche/page.tsx [hunk 5]
```diff
@@ -144,7 +144,7 @@ export default function About() {
             propre API pour son moteur de recherche et la met également à
             disposition gratuitement :{' '}
             <a
-              href="https://api.gouv.fr/les-api/api-recherche-entreprises"
+              href="https://www.data.gouv.fr/fr/dataservices/api-recherche-entreprises"
```
Comment (HAEKADI) [inline app/(header-default)/a-propos/comment-ca-marche/page.tsx:147]:
```suggestion
              href="https://www.data.gouv.fr/fr/dataservices/api-recherche-dentreprises/"
```

**Review by HAEKADI — CHANGES_REQUESTED** [review]
Most of the links don't work 😆

File: data/administrations/dila.yml [hunk 6]
```diff
@@ -7,15 +7,15 @@ contact: https://www.dila.premier-ministre.gouv.fr/contacts/nous-contacter
 apiMonitors:
   - apiName: API BODACC
     apiSlug: bodacc
-    apiDocumentationLink: https://api.gouv.fr/les-api/api-bulletin-annonces-civiles-commerciales-bodacc
+    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-bulletin-annonces-civiles-commerciales-bodacc
```
Comment (HAEKADI) [inline data/administrations/dila.yml:10]:
```suggestion
    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-bulletin-officiel-des-annonces-civiles-et-commerciales-bodacc/
```

File: data/administrations/dila.yml [hunk 7]
```diff
@@ -7,15 +7,15 @@ contact: https://www.dila.premier-ministre.gouv.fr/contacts/nous-contacter
...
     updownIoId: wd04
   - apiName: API Annonces et comptes annuels des associations (JOAFE)
     apiSlug: joafe
-    apiDocumentationLink: https://api.gouv.fr/les-api/api-annonces-comptes-annuels-associations-joafe
+    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-annonces-comptes-annuels-associations-joafe
```
Comment (HAEKADI) [inline data/administrations/dila.yml:14]:
```suggestion
    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-annonces-officielles-et-comptes-annuels-des-associations/
```

File: data/administrations/dila.yml [hunk 8]
```diff
@@ -7,15 +7,15 @@ contact: https://www.dila.premier-ministre.gouv.fr/contacts/nous-contacter
...
     updownIoId: 8pjy
   - apiName: API Annuaire des services publics
     apiSlug: annuaire-administration
-    apiDocumentationLink: https://api.gouv.fr/les-api/api-annuaire-administration-services-publics
+    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-annuaire-administration-services-publics
```
Comment (HAEKADI) [inline data/administrations/dila.yml:18]:
```suggestion
    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-annuaire-de-ladministration-et-des-services-publics/
```

File: data/administrations/marche-inclusion.yml [hunk 9]
```diff
@@ -6,7 +6,7 @@ logoType: paysage
 contact: https://lemarche.inclusion.beta.gouv.fr/contact/
 apiMonitors:
   - apiName: API Marché de l’Inclusion
-    apigouvlink: https://api.gouv.fr/les-api/api-structures-inclusion
+    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-structures-inclusion
```
Comment (HAEKADI) [inline data/administrations/marche-inclusion.yml:9]:
```suggestion
    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-marche-de-linclusion/
```

File: data/administrations/insee.yml [hunk 10]
```diff
@@ -5,7 +5,7 @@ logoType: portrait
 long: Institut national de la statistique et des études économiques (Insee)
 contact: https://www.insee.fr/fr/information/1302169
 apiMonitors:
-  - apiDocumentationLink: https://api.gouv.fr/les-api/sirene_v3
+  - apiDocumentationLink: https://www.data.gouv.fr/fr/api-sirene
```
Comment (HAEKADI) [inline data/administrations/insee.yml:8]:
```suggestion
  - apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-sirene/
```

**Review by HAEKADI — CHANGES_REQUESTED** [review]

File: data/administrations/dinum.yml [hunk 11]
```diff
@@ -13,11 +13,11 @@ apiMonitors:
     updownIoId: fpag
   - apiName: API Recherche d’entreprises
     apiSlug: recherche-entreprise
-    apiDocumentationLink: https://api.gouv.fr/les-api/api-recherche-entreprises
+    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-recherche-entreprises
```
Comment (HAEKADI) [inline data/administrations/dinum.yml:16]:
```suggestion
    apiDocumentationLink: https://www.data.gouv.fr/fr/dataservices/api-recherche-dentreprises/
```

**Comment by XavierJp** [comment]
I feel like every links has changed, making it very difficult for the api and data team to ever create redirections

File: app/(header-default)/donnees/api-entreprises/page.tsx [hunk 12]
```diff
@@ -103,7 +103,11 @@ export default function AccesByAPIPage() {
       </p>
       <ul>
         <li>
-          l’<a href="https://api.gouv.fr/les-api/sirene_v3">API Sirene</a> de l’
+          l’
+          <a href="https://www.data.gouv.fr/fr/dataservices/api-sirene-1/">
```
Comment (HAEKADI) [inline app/(header-default)/donnees/api-entreprises/page.tsx:107]:
```suggestion
          <a href="https://www.data.gouv.fr/fr/dataservices/api-sirene/">
```

**Review by HAEKADI — APPROVED** [review]

**Pull request merged**